### PR TITLE
Update XCode to 10.2.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Then, build the compiler and libraries:
 
 To build with experimental targets support compile with `-Porg.jetbrains.kotlin.native.experimentalTargets`.
 
-The build can take about an hour on a Macbook Pro.
+./gradlew dist -PsharedProjectPathThe build can take about an hour on a Macbook Pro.
```
Comment (SvyatoslavScherbina) [inline README.md]: ?

File: konan/konan.properties [hunk 2]
```diff
@@ -25,6 +25,7 @@ dependencyProfiles = default alt
 airplaneMode = false
 # if true, ignores Xcode version check.
 ignoreXcodeVersionCheck = false
+useFixedXcodeVersion = 10.2
```
Comment (SvyatoslavScherbina) [inline konan/konan.properties]: Doesn't this force a user to use Xcode 10.2?

File: buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt [hunk 3]
```diff
@@ -85,9 +85,21 @@ open class FrameworkTest : DefaultTask() {
     }
 
     private fun runTest(testExecutable: Path) {
+        val target = project.testTarget()
+        val platform = project.platformManager().platform(target)
+        assert(platform.configurables is AppleConfigurables)
```
Comment (SvyatoslavScherbina) [inline buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt]: What's the motivation behind asserting the condition to be checked by `as` at next line?

File: buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt [hunk 4]
```diff
@@ -85,9 +85,21 @@ open class FrameworkTest : DefaultTask() {
...
+        val configs = platform.configurables as AppleConfigurables
+        val swiftPlatform = when (target) {
+            KonanTarget.IOS_X64 -> "iphoneossimulator"
```
Comment (SvyatoslavScherbina) [inline buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt]: `iphonesimulator`?

File: konan/konan.properties (hunk 2 above)
Comment (olonho) [reply konan/konan.properties]: Maybe filter that out in release bundle.

File: buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt (hunk 3 above)
Comment (olonho) [reply buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt]: Not much, I guess.

File: konan/konan.properties (hunk 2 above)
Comment (olonho) [reply konan/konan.properties]: Not sure if I like approach taken here, let's discuss.

**Pull request merged**